pkg/observ/stats.go

```diff
@@ -0,0 +1,84 @@
+package observ
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/errors"
+	"go.opencensus.io/exporter/prometheus"
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/stats/view"
+)
+
+// RegisterStatsExporter determines the type of StatsExporter service for exporting stats from Opencensus
+// Currently it supports: prometheus
+func RegisterStatsExporter(app *buffalo.App, statsExporter, service string) (func(), error) {
+	const op errors.Op = "RegisterStatsExporter"
```

Comment:
Can you add the pkg name like `observ.RegisterStatsExporter` ?

Reply:
sure, I was keeping convention from second obervable packet. Should I change there also?

Reply:
I guess yes so I added :slightly_smiling_face: